supabase/config.toml

```diff
@@ -276,3 +276,29 @@ s3_region = "env(S3_REGION)"
 s3_access_key = "env(S3_ACCESS_KEY)"
 # Configures AWS_SECRET_ACCESS_KEY for S3 bucket
 s3_secret_key = "env(S3_SECRET_KEY)"
+
+# CONFIGS FOR PERSISTENT BRANCHES BELLOW:
+# Settings above are applied to local enviroment and hosted preview environments (feature branches) that are ephemeral.
+# For persitent branches like `master` we need to use [remotes]. See the docs here
+# https://supabase.com/docs/guides/deployment/branching?queryGroups=platform&platform=existing#branch-configuration-with-remotes
+
+# Master configs (live enviroment):
+# we are only specifying overriden defaults here for now
+[remotes.master]
+project_id = "eepvxhmgmtasvfzwlrbh"
+
+#[remotes.master.api]
```

Comment:
first I will try if just specifying:
```
[remotes.master]
project_id = "eepvxhmgmtasvfzwlrbh"
```

will be correct. if not I will uncomment the section below and see if that helps

Reply:
actually I will try this first https://github.com/MakePrisms/boardwalkcash/pull/281

Reply:
> actually I will try this first https://github.com/MakePrisms/boardwalkcash/pull/281

this didn't work. will try with this pr now